BoyumFoosballStats/Components/Charts/ChartComponentBase.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using BoyumFoosballStats.Components.Charts.Models;
+using Microsoft.AspNetCore.Components;
+
+namespace BoyumFoosballStats.Components.Charts;
+
+public class ChartComponentBase : ComponentBase
+{
+    [Parameter] 
+    public List<ChartDataItem> Data { get; set; }
+    
+    protected static string FormatAsPercentage(object value)
+    {
+        return $"{Convert.ToDouble(value) * 100:0.##}%";
```

Comment:
I made an extension method for the same purpose. It might make sense to get rid of one of them at some point.
You can have a look and see which usage you prefer